easybuild/framework/easyconfig/types.py

```diff
@@ -133,6 +133,18 @@ def to_toolchain(tcspec):
     return res
 
 
+def to_osdependencies(os_dep_specs):
```

Comment:
this can probably be more generic: `to_list_of_strings_and_tuples`, since we may also need this for the values in `sanity_check_paths`